Change Rust Hack & Learn

skade opened the pull request:
This changes the Rust activity to an instruction to set up a meetup.

It's still WIP, especially around which links to give to the organisers.

skade commented:
@MichaelKohler clerical error, thanks for the notice.

I intend to update it before a merge, I still want to have people from the Rust content team to review it and give a few pointers to beginners material.

skade commented:
@MichaelKohler @badboy @ashleygwilliams Needs final review, along with https://github.com/rust-community/events-team/pull/9/files

🚢ing ! \o/

MichaelKohler commented:
I will review this later today. I'd also like @couci to review this.

MichaelKohler commented on _pages/rust-hack.md:
```diff
@@ -49,41 +55,44 @@ Rust is a systems programming language that runs blazingly fast, prevents segfau
+* Help people that don't have a project to find one
+* Open time: people should just work on whatever they have. Some people will just chat, make sure they don't disturb others.
+* Closing: go through the group again and ask people for their progress.
+  - If time allows, make a [Show \& Tell](https://en.wikipedia.org/wiki/Show_and_tell_(education)) and have people actually present their learnings in a very short talk.
```
This renders as "Show \\& Tell" ;)

couci commented on _pages/id/rust-hack.md:
```diff
@@ -111,7 +97,7 @@ Additional to the recommended event flow here are some things we’d like you, t
 
 ## Extra
-While the focus of this activity should be on these events, there are other ways to stay more involved with the Rust community after.
+While the focus of this activity should be on these events, people can get more involved more involved with the Rust community after.
```
I guess here it's "people can get more involved with the Rust community after

couci commented on _pages/rust-hack.md:
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
-Rust will also be used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+Rust is used extensively for Mozilla projects -- including this year as Servo (written in Rust) is advanced even further, and parts of it are moved into Gecko/Firefox. Having more Mozilla contributors know Rust will give them a chance to contribute to the most exciting and important technical projects coming up for Mozilla.
+
+Rust is also a good project to be engaged in for people that don't want or can't participate in a Mozilla project, but like to be involved with something close.
```
I think it's ok. While activate is organized mainly by Mission Driven Mozillians, thus their main objective is to bring more people to Mozilla, it's ok to mention that Rust is also a stand alone thing

skade replied:
"Rust is also a good subject to get people closer to Mozilla gradually, as it has smaller project sizes and potential for individual contribution - for example by writing a useful library."

couci commented on _pages/rust-hack.md:
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
+* 1,000 programmers are introduced to and are well on their way to learning Rust
+* 10 ongoing Rust Hack and Learn events
+* 500 social media shares
+* Each organizer schedules 3 Rust Hack and Learn events
```
I understand this as a whole activity goal and not per person

skade replied:
That's also how I understood it from the other activities.

couci commented on _pages/rust-hack.md:
```diff
@@ -49,41 +55,44 @@ Rust is a systems programming language that runs blazingly fast, prevents segfau
 ## Activity Format
 
-Get your community behind this exciting new programming language by running a learning event.
+For duration, we recommend at least 2-3 hours event with a minimum of 10 attendees.
```
"a 2-3 hours event"

couci commented:
thanks @skade for all the work here, except the minor stuff mentioned this looks good to me

skade commented on _pages/rust-hack.md:
```diff
@@ -15,30 +15,36 @@ redirect_from: "/developer-engagement/rust-hack/"
+
+Rust is also a good project to be engaged in for people that don't want or can't participate in a Mozilla project, but like to be involved with something close.
 
 ### Goals for this area
```
What would the measurement period be? "In 2018?"

skade commented:
Pushed a polished version, /cc @couci @MichaelKohler

samim111 commented:
sscccssxx

On 4/1/18, Michael Kohler <notifications@github.com> wrote:
> MichaelKohler approved this pull request.
>
> Thanks for the fixes! This looks good to me now. @skade would you like to
> write a short abstract on the new format and the reasons for the rewrite, so
> we could include that in a Discourse post for the announcement once we merge
> this?
>
>
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mozilla/activate.mozilla.community/pull/293#pullrequestreview-108517447

skade commented:
@MichaelKohler 

> We've rewritten the Rust activity, as a lot of it was outdated. We also changed its nature a little. It now describes how to kick off a regular meeting, in the style of a "Hack & Learn". This allows you to run the event without having a speaker close by or without having a lot of Rust experience. The goals have changed a little. Another change was to get the activity a little more integrated with the Rust community, so that they can assist you with you questions and offer promotion. It should now also be easier to run and setup.

MichaelKohler commented:
@skade thanks a lot!

@couci wanna do a final sign-off on this? Once done, I can merge it and then announce it on Discourse / reps-general and Telegram.

couci commented:
lgtm thanks a lot @skade !

MichaelKohler commented:
Thanks for the review. I guess this is ready to be merged then! I will do so and send out the communication by end of the weekend at latest.
